app/Http/Livewire/BlockTable.php

```diff
@@ -5,23 +5,42 @@
 namespace App\Http\Livewire;
 
 use App\Models\Block;
-use App\Models\Scopes\OrderByHeightScope;
+use App\Models\Scopes\OrderByTimestampScope;
 use App\ViewModels\ViewModelFactory;
 use ARKEcosystem\Foundation\UserInterface\Http\Livewire\Concerns\HasPagination;
+use Illuminate\Pagination\LengthAwarePaginator;
 use Illuminate\View\View;
 use Livewire\Component;
 
 final class BlockTable extends Component
 {
     use HasPagination;
 
+    public const PER_PAGE = 15;
+
     /** @var mixed */
     protected $listeners = ['currencyChanged' => '$refresh'];
 
     public function render(): View
     {
+        $lastBlock = Block::withScope(OrderByTimestampScope::class)->first();
+
+        $lastBlockHeight = $lastBlock->height;
+        $heightTo        = $lastBlockHeight->minus(self::PER_PAGE * ($this->page - 1))->toNumber();
+        $heightFrom      = $lastBlockHeight->minus(self::PER_PAGE)->toNumber();
```

Comment:
I'm having some issues with the later pages, e.g. `/blocks?page=1645047` results in pagination only show up to page 100 (while it showed that page as last page before) and the actual data shown is a height ~1500 while it should have been the first 15 blocks 🤔

Reply:
:thinking: